General/general_shacl.ttl

```diff
@@ -0,0 +1,131 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+general:GeneralShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node gx:DataResource ;
+            sh:order 1 ;
+            sh:path general:DataResource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:LinksShape ;
+            sh:order 4 ;
+            sh:path general:Links ] ;
+    sh:targetClass general:General .
+
+general:LinksShape a sh:NodeShape ;
+    sh:property [ skos:example "link to screenshot, video, routing" ;
+            sh:description "Reference to screenshot data"@en ;
+            sh:message "Validation of media failed!"@en ;
+            sh:name "media"@en ;
+            sh:node general:LinkShape ;
+            sh:order 2 ;
+            sh:path general:media ],
+        [ skos:example "???" ;
+            sh:datatype xsd:string ;
+            sh:description "contract id -> is generated/assigned later via the EDC"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of contractID failed!"@en ;
+            sh:name "contractID"@en ;
+            sh:order 3 ;
+            sh:path general:contractID ],
```

Comment:
should we not write contractId (and not contractID)?
I would put it into the General root section and not inside the LinksShape (since contractId is no link).

Reply:
sounds plausible - please discuss with the others and adapt. But as mentioned above, there is almost nothing left in General Shacl.

Reply:
@jtdemer Also here a discussion point for the DataProvider's group.

Reply:
add contractID in sub category data

Reply:
ok, resolved